Game.cpp

```diff
@@ -0,0 +1,43 @@
+#include "Game.h"
+
+void Game::setAttacker() {
+    if (round % 2 == 0) {
+        att = units.at(1);
+        def = units.at(0);
+    }
+    else {
+        att = units.at(0);
+        def = units.at(1);
+    }
+    cout << att->getName() << " -> " << def->getName() << endl;
```

Comment:
ha a kodba be van egetve, hogy 2 unit van, akkor minek egy full vectort felvenni ra?